### ISSUE TITLE
Colored Wool depending on team and items on spawn

### ISSUE THREAD

**Issue by FleeingChickens** [Misat11/BedWars#11]
Would it be possible to add a wool block that you could buy and that would change color depending on the color of your team? Such as if you were on the red team and you bought wool, it would be red wool, or if you were on the blue team and bought wool it would be blue wool.

Also is it possible to add items that you get on spawn?
Such as a sword and armor at the start of the game and on respawn?

**Comment by FleeingChickens** [comment]
Also i forgot to add that with the items on spawn that you would be able to configure it in the config or something

**Comment by Misat11** [comment]
d696d44 Colored wool will be in next version **0.1.0**, now available in development version of 0.1.0 from Jenkins.
```yaml
  - price: 1
    price-type: bronze
    stack:
      ==: org.bukkit.inventory.ItemStack
      v: 1631
      type: WHITE_WOOL
      amount: 2
    properties: # You must add properties
    - name: "transform::applyColorByTeam" # and this property, now working just for wool
```

**Comment by Misat11** [comment]
9d2ec11dc17b076c72f62fd45d6d4520877c7a6c On Game Start and On Player Respawn items are added in development versions of **0.1.3**
